### PR TITLE
[DOCU-1615] rewrite Konnect Developer Registration page

### PR DESCRIPTION
**Pull request by falondarville**
### Review

@lena-larionova 

### Summary

* rewrite Konnect Developer Registration page
* remove login page image

### Reason

Previous text was confusing and hard to follow. It also over-explained simple tasks like filling out the registration form, while skipping over more meaningful information. 

### Testing

https://deploy-preview-3078--kongdocs.netlify.app/konnect/dev-portal/developers/dev-reg/

### CODE REVIEW

File: app/konnect/dev-portal/developers/dev-reg.md [hunk 1]
```diff
@@ -1,51 +1,27 @@
...
+5. At the bottom of the login form, click **Sign Up**. 
 
-   ![Konnect Developer Registration](/assets/images/docs/konnect/konnect-dev-sign-up.png)
+6. Fill out the registration form and click **Create Account**. 
 
-3. Complete the {{site.konnect_short_name}} Developer dialog:
+7. Check your email for registration confirmation. 
 
-   1. Enter your **First Name** and **Last Name**.
+8. Once your email is confirmed, you _will not be able_ to log in until an admin approves your request. However, if the admin has set the Konnect account to [auto approve](/konnect/dev-portal/administrators/auto-approve-devs-apps) developer registrations, you will be able to login immediately.
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/konnect/dev-portal/developers/dev-reg.md:25]: This information isn't truly a step, so you should either roll it into the previous step or the next one.

File: app/konnect/dev-portal/developers/dev-reg.md [hunk 2]
```diff
@@ -1,51 +1,27 @@
...
-   5. Check your email for registration confirmation. In the confirm your address email, click the confirmation link.
-
-   6. A message indicates your email has been confirmed, and your account is awaiting approval
-      if [auto approve](/konnect/dev-portal/administrators/auto-approve-devs-apps) is not enabled.
-
-6. After your request has been [approved by an admin](/konnect/dev-portal/administrators/manage-devs) or automatically approved,
-   click **Login**.
-
-7. Enter your credentials and log in. If you experience any access issues, contact your {{site.konnect_short_name}} admin.
+9. Once approved, log in using your Portal credentials. Contact your {{site.konnect_short_name}} admin if you have login issues.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/developers/dev-reg.md:27]:
```suggestion
9. Log in using your Portal credentials. Contact your {{site.konnect_short_name}} admin if you have login issues.
```
Unfortunately, currently they have no way of knowing when they're been approved, right :( It might be good to move all of step 8 into this step: try to log in, oh you can't? that means auto-approve isn't on and you have to wait for your admin.

File: app/konnect/dev-portal/developers/dev-reg.md [hunk 3]
```diff
@@ -1,51 +1,38 @@
...
-      The password must be a minimum of 8 characters long and contain at least two of the
-      following: a lowercase letter, an uppercase letter, or a special
-      character.
+3. Fill out the registration form and click **Create Account**. 
 
-   4. Click **Create Account**.
+4. Check your email for registration confirmation. 
 
-   5. Check your email for registration confirmation. In the confirm your address email, click the confirmation link.
+5. After you have confirmed your email, you _will not be able_ to log in, unless  [auto approve](/konnect/dev-portal/administrators/auto-approve-devs-apps) is enabled by a {{site.konnect_short_name}} service admin or organization admin. If you try to log in right after creating the account and receive an error message, it's likely that the admin hasn't approved your account yet.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/developers/dev-reg.md:36]:
```suggestion
5. After you have confirmed your email and received approval from your admin, log 
into the Dev Portal.
    
    If your admin has auto-approve enabled, you can log in right after confirming 
    your email. Otherwise, you _will not be able_ to log in until your admin approves 
    the request. If you try to log in right after creating the account and receive an 
    error message, it's likely that the admin hasn't approved your account yet.
```

**Review by HCloward — APPROVED** [review]
Tested all links and reviewed all content. Looks good to me.

**Review by lena-larionova — APPROVED** [review]
LGTM, nice work!
